src/config/dynamic-theme-fixes.config

```
@@ -8423,7 +8423,7 @@ CSS
 oalevelsolutions.com
 
 INVERT
-.WordSection1 > p > span > img
+* > p > span > img
```

```suggestion
p > span > img
```

No need to specify `* >`